bundle.Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM scratch
 
+ARG BUNDLEVERSION=1.9.0
```

Comment:
we'll need to update this every time we cut the branch, correct? could we please document somewhere?

Reply:
1.9.0 value here si just the default one if the arg is not specified.

In the Makefile we already have a variable with the current version that is updated every new version, we set the argument using this variable, so this should not add any work when cutting branch.

Reply:
maybe just remove the default value then ? So there's no ambiguity